### PR TITLE
Fixes #79 - Add support for 3 input inverters

### LINKED ISSUE #79: Support for inverters with 3 PV inputs
Add support for SMA inverters with 3 PV string inputs. Initially reported at https://github.com/home-assistant/core/issues/52850 - sorry about that, didn't realize `pysma` was it's own repository. Pull request coming shortly.

### PR DESCRIPTION
**Pull request by scotte**
Adds support for SMA inverters with 3 string inputs.

### CODE REVIEW

File: pysma/definitions.py [hunk 1]
```diff
@@ -61,14 +61,20 @@
 pv_power_a = Sensor("6380_40251E00_0", "pv_power_a", unit="W")
 #: Current power generated by the solar panels (B side)
 pv_power_b = Sensor("6380_40251E00_1", "pv_power_b", unit="W")
+#: Current power generated by the solar panels (C side)
+pv_power_c = Sensor("6380_40251E00_2", "pv_power_c", unit="W")
```
Comment (rklomp) [inline pysma/definitions.py:65]:
```suggestion
pv_power_c = Sensor("6380_40251E00_2", "pv_power_c", unit="W", enabled=False)
```

File: pysma/definitions.py [hunk 2]
```diff
@@ -61,14 +61,20 @@
...
 #: Current voltage generated by the solar panels (A side)
 pv_voltage_a = Sensor("6380_40451F00_0", "pv_voltage_a", unit="V", factor=100)
 #: Current voltage generated by the solar panels (B side)
 pv_voltage_b = Sensor("6380_40451F00_1", "pv_voltage_b", unit="V", factor=100)
+#: Current voltage generated by the solar panels (C side)
+pv_voltage_c = Sensor("6380_40451F00_2", "pv_voltage_c", unit="V", factor=100)
```
Comment (rklomp) [inline pysma/definitions.py:71]:
```suggestion
pv_voltage_c = Sensor("6380_40451F00_2", "pv_voltage_c", unit="V", factor=100, enabled=False)
```

File: pysma/definitions.py [hunk 3]
```diff
@@ -61,14 +61,20 @@
...
 #: Current amperage generated by the solar panels (A side)
 pv_current_a = Sensor("6380_40452100_0", "pv_current_a", unit="A", factor=1000)
 #: Current amperage generated by the solar panels (B side)
 pv_current_b = Sensor("6380_40452100_1", "pv_current_b", unit="A", factor=1000)
+#: Current amperage generated by the solar panels (C side)
+pv_current_c = Sensor("6380_40452100_2", "pv_current_c", unit="A", factor=1000)
```
Comment (rklomp) [inline pysma/definitions.py:77]:
```suggestion
pv_current_c = Sensor("6380_40452100_2", "pv_current_c", unit="A", factor=1000, enabled=False)
```

**Review by rklomp — CHANGES_REQUESTED** [review]
I think we should set `enabled=False` to have them disabled by default in home assistant. If a user has a third string they could enable the sensors themselves.

File: pysma/definitions.py (hunk 1 above)
Comment (scotte) [reply pysma/definitions.py:65]: That's fine, although I don't really see this any different than other sensors which are enabled by default and not all inverters have. For example, my inverter doesn't support 11 of the existing sensors that are enabled by default and which I disable manually.

I suspect the current set is probably based around what your own inverter supports. And that's fine, but I'm not sure that translates to everyone any better. I would argue it's easier to have them all enabled by default and let people turn off the ones that don't apply to them than it is to have sensors disabled and try and figure out which ones are needed. I also understand you own this integration so you are free to choose the defaults you want. :-)

Thanks!

Comment (rklomp) [reply pysma/definitions.py:65]: I've by default disabled quite a few sensors in #78 as requested by the HA team.

Comment (scotte) [reply pysma/definitions.py]: I've updated the PR with the sensors I added disabled by default.

Even with #78, there are many such as everything having to do with L3, or most of the yield sensors, which don't apply for my particular installation and/or model. But, I am not trying to have an argument, just mention that as with all defaults, there is no such thing as good defaults. :-) Thanks again!

**Comment by scotte** [comment]
And another commit to (hopefully) make linting happy.

**Review by rklomp — APPROVED** [review]

**Comment by rklomp** [comment]
I have released version 0.6.3.
If you want you can make a pull request for HA as well to bump the version in the manifest file. Otherwise I will do that later this week.

**Pull request merged**